README.md

```diff
@@ -15,32 +17,59 @@ The command line interface (CLI) allows users to submit their project package de
 
 [![asciicast](https://asciinema.org/a/431262.svg)](https://asciinema.org/a/431262)
 
-# Quickstart for Linux or macOS
-1. Download and unzip the latest [release package](https://github.com/phylum-dev/cli/releases/latest/download/phylum-cli-release.zip)
-2. Run the installer script
-```
-./install.sh
-```
-3. [Register](https://docs.phylum.io/docs/authentication) for an account (if you don't already have one)
-```
-phylum auth register
-```
-4. [Authenticate](https://docs.phylum.io/docs/authentication) with Phylum (requires [activation](https://github.com/phylum-dev/cli/tree/development#activation) by Phylum)
-```
-phylum auth login
-```
-5. [Create a new Phylum project](https://docs.phylum.io/docs/projects#creating-a-new-project) in your project directory
-```
-phylum projects create <project-name>
-```
-6. [Submit your package lock file](https://docs.phylum.io/docs/analyzing-dependencies)
-```
-phylum analyze <package-lock-file.ext>
-```
+## Quickstart for Linux or macOS
```

Comment:
The [User docs](https://docs.phylum.io/v1.3/docs/getting-started) were updated to match. Well...a suggested edit was submitted. It would be great if *all* the docs were co-located with the code, so that updates can be made, reviewed, pushed, and published synchronously.

Reply:
Chatted with @furi0us333 about this yesterday. Automating the docs would be wonderful.

Reply:
I am working on this and hope to have a solution soon.